macros/edr/materializations/tests/handle_failed_test_result_count.sql

```diff
@@ -0,0 +1,63 @@
+{% macro handle_failed_test_result_count(flattened_test) %}
+  {% set result_count_query = elementary.get_failed_test_result_count_query(flattened_test) %}
+  {% if result_count_query %}
+    {% set result_count = run_query(result_count_query).columns[0].values()[0] %}  
+  {% else %}
+    {% set result_count = none %}
```

Comment:
Is there a point in caching the result count if it's `null`?
Maybe just not add it there?

Reply:
On it